From a4259b9799ebb0f31223d52def871f4851217315 Mon Sep 17 00:00:00 2001
From: Artem Senichev <a.senichev@yadro.com>
Date: Tue, 13 Mar 2018 14:50:56 +0300
Subject: [PATCH] Increase default watchdog timeout to 300 seconds

We need more than 2 minutes to go through some IPL, for instance
step "13.10 mss_draminit_trainadv" tooks 215 seconds to initialize 8 TiB RAM.

Signed-off-by: Artem Senichev <a.senichev@yadro.com>
---
 src/include/usr/ipmi/ipmiwatchdog.H | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/usr/ipmi/ipmiwatchdog.H b/src/include/usr/ipmi/ipmiwatchdog.H
index d730e6bc4..37f73624b 100644
--- a/src/include/usr/ipmi/ipmiwatchdog.H
+++ b/src/include/usr/ipmi/ipmiwatchdog.H
@@ -58,7 +58,7 @@ namespace IPMIWATCHDOG
 #ifdef CONFIG_BMC_IPMI_LONG_WATCHDOG
 const uint16_t  DEFAULT_WATCHDOG_COUNTDOWN = 2400;
 #else
-const uint16_t  DEFAULT_WATCHDOG_COUNTDOWN = 120;
+const uint16_t  DEFAULT_WATCHDOG_COUNTDOWN = 300;
 #endif
 
 /**
-- 
2.14.1

